packages/api/package.json

```diff
@@ -38,6 +38,7 @@
     "swagger"
   ],
   "dependencies": {
+    "@readme/api-core": "file:../core",
```

Comment:
we grab the package version from `@readme/api-core`, which is why it's now a regular dep (but will defer to you on the best way to bring this in)